.github/workflows/reusable-verify.yml

```diff
@@ -93,6 +100,32 @@ jobs:
           squidex-ci-client-id: ${{ steps.setup.outputs.crn-squidex-ci-client-id }}
           squidex-ci-client-secret: ${{ secrets.CRN_SQUIDEX_CI_CLIENT_SECRET }}
 
+  crn-contentful-schema:
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment-name }}
+    container:
+      image: ghcr.io/yldio/asap-hub/node-python-sq:be0ebbcbd0ff209d56070f09590c5d9622e8a6dc
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup
+        id: setup
+        uses: ./.github/actions/setup-environment
+        with:
+          environment-name: ${{ inputs.environment-name }}
```

Comment:
is this needed even if we add `setup` in here?

```
  verify:
    needs: [setup, deployment]
```

Reply:
it is, at the on-push-branch level we do a different setup, one which runs all the contentful checks (which the one above does not), see:
https://github.com/yldio/asap-hub/blob/4adc41c58b228c739bf69999e6923d8102155904/.github/workflows/on-push-branch.yml#L27